### PR TITLE
Integration tests

### PR DESCRIPTION
**Pull request by lawlawlaw**

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -9,3 +9,6 @@ kms = { git = "https://github.com/commerceblock/kms-secp256k1" }
 
 [patch.crates-io]
 rust-gmp = { version = "0.5.0", features = ["serde_support"], git = "https://github.com/KZen-networks/rust-gmp" }
+
+#[development]
```
Comment (tomossomot) [inline Cargo.toml:13]: Should this be here?

File: integration-tests/Cargo.toml [hunk 2]
```diff
@@ -42,3 +42,6 @@ path = "../client"
 
 [dependencies.shared]
 path = "../shared"
+
+#[development]
```
Comment (tomossomot) [inline integration-tests/Cargo.toml:46]: and this

File: server/src/lib.rs [hunk 3]
```diff
@@ -77,14 +74,17 @@ pub struct DatabaseW(postgres::Connection);
 pub struct DatabaseR(postgres::Connection);
 
 pub struct PGDatabase {
-    pub db_connection: fn() -> r2d2::PooledConnection<PostgresConnectionManager>,
+    //pub con_man: PostgresConnectionManager,
```
Comment (tomossomot) [inline server/src/lib.rs:77]: rm these

File: server/src/storage/db.rs [hunk 4]
```diff
@@ -149,27 +149,23 @@ impl Column {
...
+        let url : String = rocket_url.clone().to_string();
+        let manager = PostgresConnectionManager::new(url, TlsMode::None).unwrap();
+        r2d2::Pool::new(manager).unwrap()
     }
 
-    fn database_w(&self) -> DatabaseW {
-        DatabaseW((self.db_connection)())
+    fn database_r(&self) -> DatabaseR {
+        DatabaseR(self.pool.get().unwrap())
+        //DatabaseR((self.db_connection)(&self.rocket_url))
```
Comment (tomossomot) [inline server/src/storage/db.rs:162]: rm

File: shared/Cargo.toml [hunk 5]
```diff
@@ -49,3 +49,7 @@ tag = "v0.2.0"
 [dependencies.multi-party-ecdsa]
 git = "https://github.com/KZen-networks/multi-party-ecdsa"
 tag = "v0.2.5"
+
+
+#[development]
```
Comment (tomossomot) [inline shared/Cargo.toml:54]: rm

File: Cargo.toml (hunk 1 above)
Comment (lawlawlaw) [reply Cargo.toml]: Removed.

File: integration-tests/Cargo.toml (hunk 2 above)
Comment (lawlawlaw) [reply integration-tests/Cargo.toml:46]: Removed.

File: server/src/lib.rs (hunk 3 above)
Comment (lawlawlaw) [reply server/src/lib.rs]: Done.

File: server/src/storage/db.rs (hunk 4 above)
Comment (lawlawlaw) [reply server/src/storage/db.rs]: Done.

File: shared/Cargo.toml (hunk 5 above)
Comment (lawlawlaw) [reply shared/Cargo.toml:54]: Done.

**Pull request merged**